app/models/User.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, **kvargs):
         self.full_name = kvargs.get('full_name')
         self.email = kvargs.get('email')
         self.password = kvargs.get('password')
-        self.is_active = kvargs.get('is _active')
+        self.is_active = kvargs.get('is_active')
         self.avatar = kvargs.get('avatar')
         self.role_id = kvargs.get('role_id')
 
```

Comment:
Owh.. it may a coflict here, because i change something in user model. We'll resolve it may be in manual mode :)